HHS/OPRE-OPS
# feat: ✨ adds AgreementMetaAccordion component

## What changed
feat: ✨ adds AgreementMetaAccordion component

## Issue

#1548 

## How to test

1. Goto Agreements list
2. Select Agreement airplane icon
3. See Agreement meta accordion
4. 💰 profit

## Screenshots

https://github.com/HHS/OPRE-OPS/assets/4629398/206bef56-9995-4559-ad39-76a0cb774cd6

## Definition of Done Checklist
